reporters/GlobalSummaryReporter.js

```diff
@@ -103,10 +104,29 @@ class GlobalSummaryReporter {
     this.createEvents = this.empEventClient.getAllCreateEvents();
```

Comment:
Follow up PR's will remove as many `this.empEventClient.x` calls as possible in favor of `this.umaClient` queries.